### ISSUE TITLE
contrast for .wdn-subhead

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#771]
The contrast ratio for `.wdn-subhead` is currently failing contrast checks.  This is because of a recent update to html_codesniffer to correctly check rgba colors.

This issue was not found by automated testing because there is not an example that includes the .wdn-subhead class.  Suggestion while working on this:  add an example for 'bands', which contains a band for each available color and each band includes headings, text, and links.  This will greatly increase contrast checking coverage.
